sections/projects.md

```diff
@@ -765,15 +765,15 @@ retainer_budget_type                     | The budget type of the retainer.
+period_target_minutes                      | The period target in minutes (for Fixed Hours and Time & Materials retainers).                                                                                                                                                                 | Integer  
+period_rollover_minutes                    | The net amount of rollover minutes to and from the period, including the fixing of conflicted time (registered after a period is locked). Applicable for Fixed Hours retainers.                                                                 | Double   
 period_target_price                      | The period target price (for Fixed Price retainers).                                                                                                                                                                                         | Double   
 period_rollover_price                    | The net amount of rollover price to and from the period, including the fixing of conflicted time (registered after a period is locked). Applicable for Fixed Price retainers.                                                                 | Double   
-estimated_billable_hours                 | Estimated billable time for the period, either from billable task estimates or resource allocations, depending on project financial settings.                                                                                                 | Double   
-actual_billable_hours                    | Actual billable time for the period, either from billable time registered or past resource allocations, depending on project financial settings. Conflicted time (registered after a period is locked) is not included.                       | Double   
+estimated_billable_minutes                 | Estimated billable time for the period, either from billable task estimates or resource allocations, depending on project financial settings.                                                                                                 | Double   
+actual_billable_minutes                    | Actual billable time for the period, either from billable time registered or past resource allocations, depending on project financial settings. Conflicted time (registered after a period is locked) is not included.                       | Double   
 remaining_work_billable_hours            | Remaining work billable time for the period, either from time remaining on unfinished billable tasks or future resource allocations, depending on project financial settings.                                                                 | Double   
 projected_total_billable_hours           | Projected total billable time for the period when work is completed (actual billable hours plus remaining work billable hours).                                                                                                               | Double   
```

Comment:
I guess remaining_work_billable_hours and projected_total_billable_hours should also be renamed here?